.lintstagedrc.json

```
@@ -0,0 +1,7 @@
+{
+    "*": [
+        "npm ci",
+        "npx pepr format --validate-only",
```

Would suggest we check if we can change to:
```suggestion
        "npx pepr format",
```

So that things get formatted pre-commit rather than just failing the commit.